src/Hydrator/MoneyHydrator.php

```diff
@@ -20,7 +20,7 @@ class MoneyHydrator implements HydratorInterface
     public function extract($object)
     {
         return [
-            'amount' => $object->getAmount(),
+            'amount' => $object->getAmount() / 100,
```

Comment:
The purpose of `mathiasverraes/money` is to always store and work with money represented in the smallest unit (because of rouding erros).

Yeah, of course we don't want to display it in cents for our user, that's why the `$amount / 100` is a **presentation concern**, so that you must handle it in view layer. I think a custom form element does the trick.

Thoughts? @olekhy @prolic @fabiocarneiro @gabrielsch